src/analysis/retail/monk/brewmaster/CHANGELOG.tsx

```diff
@@ -1,11 +1,12 @@
 import { change, date } from 'common/changelog';
 import SPELLS from 'common/SPELLS';
 import talents from 'common/TALENTS/monk';
-import { emallson } from 'CONTRIBUTORS';
+import { emallson, Vetyst } from 'CONTRIBUTORS';
 import SpellLink from 'interface/SpellLink';
 
 // prettier-ignore
 export default [
+  change(date(2025, 6, 8), <>Add preperation section to overview.</>, Vetyst),
```

Comment:
```suggestion
  change(date(2025, 6, 8), <>Add preparation section to overview.</>, Vetyst),
```

typo (i'm doing these as comments so i can slam the github "apply suggested fix" button)